### DISCUSSION TITLE
No messages visible for required fields when accordion's groups are closed

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12068] [Feature Requests]
On accordion's groups closed containing missing required fields, when we click on Save, no messages are shown (they are inside the closed groups) and as nothing happens, the user will call the support to complain that the app is not working.

Could some thing alert the user? ideally opening only the groups containing the messages.

**Comment by rijkvanzanten** [comment]
This was actually just greatly improved in https://github.com/directus/directus/pull/12001 😄 It'll be released in the upcoming release 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
